libraries/AP_Vehicle/AP_Vehicle.h

```diff
@@ -151,12 +152,15 @@ class AP_Vehicle : public AP_HAL::HAL::Callbacks {
     // returns true if the vehicle has crashed
     virtual bool is_crashed() const;
 
+#if AP_EXTERNAL_CONTROL_ENABLED
```

Comment:
Should this not be:
```suggestion
#if AP_EXTERNAL_CONTROL_ENABLED || AP_SCRIPTING_ENABLED
```

Reply:
THe plan was that `AP_SCRIPTING_ENABLED` depends on `AP_EXTERNAL_CONTROL_ENABLED`. If you agree on that approach, then I can add some code in waf to guarantee it.

Reply:
We decided not to add this dependency quite yet and use Pete's proposed solution.